sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -299,12 +299,7 @@ def to_proto_str(element):
     return element._to_proto_str()
 
   def expand(self, pcoll):
-    if self.with_attributes:
-      pcoll = pcoll | 'ToProtobuf' >> Map(self.to_proto_str)
-
-    # Without attributes, message data is written as-is. With attributes,
-    # message data + attributes are passed as a serialized protobuf string (see
-    # ``PubsubMessage._to_proto_str`` for exact protobuf message type).
+    pcoll = pcoll | 'ToProtobuf' >> Map(self.to_proto_str)
```

Comment:
Update the pydoc for 'with_attributes' property above ?

Reply:
I just realized that `with_attributes` means different type of input. Changed the expansion with respect to `with_attributes`.